### PR TITLE
add apiextensions.k8s.io to clusterrole

### PR DESCRIPTION
**Pull request by dangula**

### CODE REVIEW

File: demo/helm/rook-operator/README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Alternatively, to deploy from a local checkout of the rook codebase (until the r
 
 ```console
 $ cd demo/helm/rook-operator
-$ helm install --name rook-operator --namespace rook .
+$ helm install --name rook-operator --namespace default .
```
Comment (bassam) [inline demo/helm/rook-operator/README.md]: @travisn why is this needed? do you know?

Comment (travisn) [inline demo/helm/rook-operator/README.md]: this should just be the namespace where the operator starts. It shouldn't matter in what namespace it starts though.

Comment (bassam) [inline demo/helm/rook-operator/README.md]: so a bug then?

Comment (travisn) [inline demo/helm/rook-operator/README.md]: no let's leave it in the rook namespace. The rest of the change looks good though.

Comment (dangula) [inline demo/helm/rook-operator/README.md]: ok, reverting back to rook namespace, smoke test passed on both rook and default namespaces.
